### PR TITLE
resolve typo

### PR DESCRIPTION
**Pull request by dofinn**

### CODE REVIEW

**Comment by dofinn** [comment]
Need to test build with go 1.13 prior to making changes as per: https://github.com/cblecker/managed-velero-operator/blob/11f47f3d35160508fa79bdfd39db977c6ff2cdf7/build/Dockerfile

**Comment by jewzaam** [comment]
@dofinn I think the Dockerfile just needs updated, see https://github.com/dofinn/splunk-forwarder-operator/pull/1 (which is a PR into this PR)

File: build/Dockerfile [hunk 1]
```diff
@@ -1,14 +1,16 @@
 FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
-ENV OPERATOR_PATH=/go/src/github.com/openshift/splunk-forwarder-operator
-COPY . ${OPERATOR_PATH}
-WORKDIR ${OPERATOR_PATH}
+#ENV OPERATOR_PATH=/go/src/github.com/openshift/splunk-forwarder-operator
+#COPY . ${OPERATOR_PATH}
```
Comment (wanghaoran1988) [inline build/Dockerfile]: I'd like remove them instead of comment out

Comment (dofinn) [reply build/Dockerfile]: done.

**Pull request merged**